ocaml/xapi-idl/network/network_interface.ml

```diff
@@ -158,7 +158,10 @@ type interface_config_t = {
   ; ipv6_conf: ipv6 [@default None6]
   ; ipv6_gateway: Unix.inet_addr option [@default None]
   ; ipv4_routes: ipv4_route_t list [@default []]
-  ; dns: Unix.inet_addr list * string list [@default [], []]
+  ; dns: (Unix.inet_addr list * string list) option [@default None]
```

Comment:
Can we assume that changing to an option type here does not change the expected JSON format? That is, existing `dns` fields in `networkd.db` (written before applying this patch) will be parsed correctly?

Reply:
Great, thanks for checking!